src/Runner/FileFilterIterator.php

```diff
@@ -52,7 +52,12 @@ public function __construct(
 
     public function accept()
     {
+        /** @var \SplFileInfo $file */
         $file = $this->current();
+        if (!($file instanceof \SplFileInfo)) {
```

Comment:
Parentheses around `instanceof` are superfluous (here and on other changed lines in this PR).

Reply:
I like the parentheses as I don't want to state 
>is `!$file` an instance of `\SplFileInfo`

even if the operator precedence changes the meaning of the sentence for me